backend/migrations/versions/025d7d620502_add_name_column_to_user_table.py

```diff
@@ -0,0 +1,30 @@
+"""Add name column to user table
+
+Revision ID: 025d7d620502
+Revises: 44b32661fe9b
+Create Date: 2020-07-31 01:23:56.055172
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '025d7d620502'
+down_revision = '44b32661fe9b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('user', sa.Column('name', sa.String(length=64), nullable=False))
+    op.create_index(op.f('ix_user_name'), 'user', ['name'], unique=True)
```

Comment:
Hm I don't think we should make this a unique indexed column since people can have the same name and there can be duplicates